setup.py

```diff
@@ -44,7 +44,11 @@
     'maintainer': [sdw, jt, ag, kh],
     # as long as 1.0.0 is not out, vsc-base should still provide vsc.fancylogger
     # setuptools must become a requirement for shared namespaces if vsc-install is removed as requirement
-    'install_requires': ['vsc-install >= %s' % VSC_INSTALL_REQ_VERSION],
+    'install_requires': [
+        'vsc-install >= %s' % VSC_INSTALL_REQ_VERSION,
+        'coloredlogs',     # automatic log colorizer
```

Comment:
need to check if this ends up in the rpm dependencies or not. but i don not see the point of listing them here since they are conditional. setuptools has something for such conditional behaviour i think
@JensTimmerman do you  rememebr the required magic for optional requirements?